test/addons/recipe/fetch

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
```

Comment:
The `Fixes: #1330` should be in the commit message, above the `Signed-off-by:` line.

The motivation that when looking at git log, you can find all the relevant issues and bugs to help maintainers understand why a change was done in the future.